config/redirects

```diff
@@ -1,6 +1,36 @@
-define: prefix docs/drivers/c
+define: prefix docs/languages/c/c-driver
 define: base https://www.mongodb.com/${prefix}
-define: versions v1.26 v1.27 1.28 master
+define: versions v1.26 v1.27 1.28 v1.29 master
 
-# raw: <source file> -> ${base}/<destination>
+symlink: current -> master
+
+raw: ${prefix}/ -> ${base}/current/
+raw: ${prefix}/stable -> ${base}/current/
+raw: ${prefix}/master -> ${base}/upcoming/
+raw: ${prefix}/current/libmongoc -> ${base}/current/
+
+(1.29-master]: ${prefix}/${version}/libmongoc/tutorials/ -> ${base}/${version}/
```

Comment:
S: did you mean to only implement this after 1.29? Otherwise, i think you should use the inclusive bracket: [

Reply:
Good catch, thanks!